sentry_sdk/integrations/django/__init__.py

```diff
@@ -325,14 +325,10 @@ def sentry_patched_drf_initial(self, request, *args, **kwargs):
 def _patch_channels():
     # type: () -> None
     try:
-        # Django < 3.0
         from channels.http import AsgiHandler  # type: ignore
     except ImportError:
-        try:
-            # DJango 3.0+
-            from django.core.handlers.asgi import ASGIHandler as AsgiHandler
```

Comment:
This was useless here. because it is also done in `_patch_django_asgi_handler()`. 
Now the code is clean again and the "import channels" is only happening when there was this `channels.http` import before, so we do not need to guard it again.